cardano-config/test/Test/Cardano/Config/Gen.hs

```diff
@@ -74,6 +80,7 @@ genShelleyGenesis =
     <*> fmap fromIntegral (Gen.word $ Range.linear 100 1000000)
     <*> fmap Map.fromList genGenesisDelegationList
     <*> fmap Map.fromList genFundsList
+    <*> pure emptyGenesisStaking
```

Comment:
Same here. I have generators in another branch, but the JSON to/from cannot be completed yet, so these tests would not pass yet.